ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,123 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin\n",
+    "\n",
+    ":::{important}\n",
+    "To convert from Kelvin to Celsius `-273.15` and to convert from Celsius to Kelvin `+273.15`\n",
+    ":::"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "6.298141316024157"
```

Comment:
Are we clearing output yet?

Reply:
Not currently, would it be a good best practice?

Reply:
I think this did change (i.e. now we clear outputs) with this PR: https://github.com/NCAR/geocat-applications/pull/102

Reply:
Looks like there are still a few outputs in this notebook as well.

Reply:
Clearing them in the most recent commit